src/Chapter1.hs

```diff
@@ -632,7 +640,8 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = let sums = (mod n 10) + (mod (div n 10) 10) in sums  
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

Note that you would need to use absolute values in order to cover negative numbers.

One hint to make your solution even shorter: you can see that you use both:

```haskell
mod (abs n) 10
div (abs n) 10
```
The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(x, y) = divMod (abs n) 10
```

You can see how we could pattern match on the pair 🙂

Reply:
I'm a C programmer. So I'm finding Haskell quite different. Will try to rectify those mistakes you told. Thank you and Have a nice day!